assembly/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../node_modules/assemblyscript/std/assembly.json",
   "include": [
-    "./**/*.ts"
+    "./**/*.ts",
+    "../node_modules/**/*/as_types.d.ts"
```

Comment:
This is an area for discussion.  I started with the `.d.ts` file because it would use node resolution to find the types file.  Here we are assume that there will a types file in the node_modules folder above.  I added this thinking it would be less noise, but now I think we should use node resolution only so that nested projects can still find the types.